lib/src/blockdev.rs

```diff
@@ -197,7 +197,7 @@ pub(crate) fn find_parent_devices(device: &str) -> Result<Vec<String>> {
         let kind = dev
             .get("TYPE")
             .with_context(|| format!("device in hierarchy of {device} missing TYPE"))?;
-        if kind == "disk" {
+        if kind == "disk" || kind == "loop" {
```

Comment:
Ah yes.  It would probably make sense to try to use a shared crate for dealing with block devices at some point.